metadata.json

```diff
@@ -11,6 +11,10 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 4.2.0 < 5.0.0"
+    },
+    {
+      "name": "puppetlabs/concat",
+      "version_requirement": ">= 1.1.1 < 2.0.0"
```

Comment:
Good question... it was just a copy/paste from another plugin. I will remove this.